hr_attendance_modification_tracking/models/hr_attendance.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Creu Blanca
+# Copyright 2021 Landoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+
+class HrAttendance(models.Model):
+    _name = "hr.attendance"
+    _inherit = ["hr.attendance", "mail.thread"]
```

Comment:
HR attendance already inherit from mail.thread from version 17.0

```suggestion
class HrAttendance(models.Model):
    _inherit = "hr.attendance"
```